### DISCUSSION TITLE
spark-operator chart

### DISCUSSION THREAD

**Discussion by quintonanderson-zetaris** [fluxcd/flux2#2683] [Q&A] [answered]
I am busy trying to deploy the [spark-operator](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator) using a HelmRepository and HelmRelease, following the mono-repo pattern. 

I have defined the HelmRepository as follows:
`
apiVersion: source.toolkit.fluxcd.io/v1beta1
kind: HelmRepository
metadata:
  name: spark-operator
spec:
  interval: 5m
  url: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator
`

While trying to synchronize I get the following error: 
`failed to download repository index: failed to cache index to temporary file: failed to fetch https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/index.yaml`

I have tried various ways to resolve the issue, including trying a GitRepository kind and pointing at the releases urls within the repo. I feel like I am missing something obvious here, any help would be appreciated.

**Comment by stefanprodan** [comment, marked as answer]
The repository URL you\'re using doesn't match what's in the readme: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator#installation it should be:

> https://googlecloudplatform.github.io/spark-on-k8s-operator
